content-src/asrouter/templates/Trailhead/Trailhead.jsx

```diff
@@ -20,41 +26,33 @@ export class Trailhead extends React.PureComponent {
     });
   }
 
-  render() {
-    const {props} = this;
-    const {content} = props.message;
+  componentDidMount() {
+    // We need to remove hide-main since we should show it underneath everything that has rendered
+    global.document.body.classList.remove("hide-main");
```

Comment:
Looks like we also needed to remove "welcome" which is used to hide overflow.

Reply:
We can probably fix this up when we add cards to top and figure out what to do with fixed search.